### PR TITLE
Search for MergExt externals bundled with the IDE

### PR DESCRIPTION
**Pull request by livecodefraser**
PR that adds the externals to the IDE bundle: https://github.com/livecode/livecode/pull/3884

### CODE REVIEW

File: Toolset/home.livecodescript [hunk 1]
```diff
@@ -1097,8 +1110,30 @@ private command revInternal__SetupExternals
...
+         
+         repeat for each line tFile in tFiles
+            put revEnvironmentMergExtPath() & "/" & tExternal & "/" & tFile & return after tExternals
+         end repeat
+      end repeat
+   end if
+   
    if there is a folder revEnvironmentUserExternalsPath() then
       set the folder to revEnvironmentUserExternalsPath()
+      set the itemDelimiter to "-"
```
Comment (livecodeali) [inline Toolset/home.livecodescript]: This isn't needed is it?

File: Toolset/home.livecodescript [hunk 2]
```diff
@@ -1,4 +1,4 @@
-script "home"
+﻿script "home"
```
Comment (livecodeali) [inline Toolset/home.livecodescript:1]: Can you check the BOM hasn't somehow gone astray? I'm never sure what's gone on when a change looks like this.

**Comment by livecodefraser** [comment]
BOM is still there. Extraneous itemDelimiter setting has been removed.

**Comment by livecodeali** [comment]
Thanks!
@livecode-vulcan review ok d79df4d

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok d79df4d14caf4ae24b545d92085c61000b502245

**Comment by livecodefraser** [comment]
Forgot to push the additions to the licenses file!

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 234048c

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 234048c0b370f8fe1876995d8fde140e387ce5ad

**Pull request merged**